packages/keycloak-user-management/src/components/CreateEditUser/tests/index.test.tsx

```diff
@@ -0,0 +1,209 @@
+import reducerRegistry from '@onaio/redux-reducer-registry';
+import { mount, shallow } from 'enzyme';
+import React from 'react';
+import { act } from 'react-dom/test-utils';
+import { history } from '@onaio/connected-reducer-registry';
+import { Provider } from 'react-redux';
+import { Router } from 'react-router';
+import store from '../../../../../../client/src/store';
+import fetch from 'jest-fetch-mock';
+import * as keycloakUserDucks from '@opensrp/store';
```

Comment:
lets move keycloak user store stuff out of the global store package

Reply:
Tracking this via https://github.com/OpenSRP/web/issues/64